### DISCUSSION TITLE
Uno Platform CreationCollisionOptions enum values do not match

### DISCUSSION THREAD

**Discussion by Avid29** [unoplatform/uno#10630] [Q&A]
In the original ``Windows.Storage.CreationCollisionOptions`` the values are
0:GenerateUnqiueName
1:ReplaceExisting
2:FailIfExists
3:OpenIfExists

In Uno, the values are
0:OpenIfExists
1:ReplaceExisting
2:GenerateUnqiueName
3:FailIfExists

Is this an intentional difference, if so why?

**Comment by jeromelaban** [comment]
Thanks for the feedback! It's definitely not intentional, though changing it now would probably be a breaking change (which we can introduce in a later version of Uno).

I'm curious though, how did you notice it?

**Reply by Avid29** [reply]
It didn't cause issues or anything. I was trying to figure out which value was the default (which I assumed would be 0) and I noticed that changed depending on which platform was selected.

**Reply by jeromelaban** [reply]
Thanks we'll take note to adjust that value in a future breaking changes round (#8339).
